### ISSUE TITLE
Taskbar icon vanishes when windows explorer is restarted.

### ISSUE THREAD

**Issue by joshuabradt** [namreeb/wowreeb#16]
This is probably not a common issue. I have to restart windows explorer every time I change my monitor configuration (I do this to enable/disable my TV as a monitor often). Whenever that happens, wowreeb's icon disappears and doesn't come back. Thus, I have to open a new instance of it each time. This doesn't happen with any other windows 10 application as far as I can tell.

**Comment by namreeb** [comment]
My guess is that there is a window message that gets sent when the icon needs to recreate itself, and that wowreeb does not listen for that window message.  I reproduced this and recorded the following window messages when explorer restarted:

```
msg: 70 (WM_VKEYTOITEM) wParam: 0 lParam: 44104700
msg: 36 (WM_GETMINMAXINFO) wParam: 0 lParam: 44103884
msg: 70 (WM_VKEYTOITEM) wParam: 0 lParam: 44104700
msg: 36 (WM_GETMINMAXINFO) wParam: 0 lParam: 44103884
msg: 49347 (0xC0C3) wParam: 0 lParam: 0
msg: 49244 (0xC05C) wParam: 0 lParam: 0
msg: 49893 (0xC2E5) wParam: 82 lParam: 34
msg: 49893 (0xC2E5) wParam: 80 lParam: 0
```

So I'll have to figure out which of these are meant to trigger the re-creation of the notification area icon.

(This is just a note for myself, you don't need to do anything more at this point).

The solution seems to be described [here](https://stackoverflow.com/questions/32044732/how-to-restore-a-missing-notification-icon-when-explorer-exe-is-restarted).

**Comment by namreeb** [comment]
Fixed in 3.4.  See binary available [here](https://github.com/namreeb/wowreeb/releases/tag/3.4).  Thanks for your feedback!

**Comment by joshuabradt** [comment]
That was quick! Works like a charm now. Thank you.
